R/wwinference.R

```diff
@@ -36,6 +36,8 @@
 #' `compile_model()`
 #' @param dist_matrix Distance matrix for spatial correlation in distance
 #' correlation function.
+#' @param bool_spatial_comp Boolean for whether or not user wants a spatial
+#' model.
```

Comment:
```suggestion
#' model. Default is `FALSE` indicating the spatial component is not being modeled
```

Reply:
Currently this is a switch for if the user wants to use a spatial model based off distances or not.  If they do want to use spatial model based off distances there is a check to ensure they have provided a distance matrix.  This can be extended later when we introduce spatial model without distance matrix.

Reply:
Do you mean that I should add another switch to which type of correlation (structured w/ dist_matrix or unstructured infer corr) and have default as structured w/ dist_matrix?  If so, I think we should wait on that for the next PR as the "frontend" will depend on how I plan to do this within the `wwinference.stan`.  But I think it will be exactly how you are suggesting.  Right now we only have the high level switch and I can improve the documentation of it.

Reply:
That sounds reasonable to me, and exactly! Fine to do it the next PR